resources/default-settings/gpio.example.yaml

```diff
@@ -1,6 +1,5 @@
 # Provides a few selected examples of GPIO configuration
 # Check out the documentation for many more configuration recipes
-# https://rpi-jukebox-rfid.readthedocs.io/en/latest/index.html
```

Comment:
We have a MD file at https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/future3/develop/documentation/builders/gpio.md